LayoutTests/platform/mac-sonoma/TestExpectations

```diff
@@ -21,4 +21,7 @@ imported/w3c/web-platform-tests/mixed-content/gen/top.http-rp/opt-in/sharedworke
 imported/w3c/web-platform-tests/mixed-content/gen/top.meta/opt-in/sharedworker-import-data.https.html [ Skip ]
 imported/w3c/web-platform-tests/mixed-content/gen/top.meta/unset/sharedworker-import-data.https.html [ Skip ]
 
+# List buttons only have tint color pre-Tahoe.
+fast/css/accent-color/datalist.html [ Pass ]
```

Comment:
Expectations in `mac-sequoia` apply to both Sequoia and Sonoma, so no change necessary here.